web/src/main/java/service/UserService.java

```diff
@@ -0,0 +1,10 @@
+package service;
+
+import org.springframework.security.core.userdetails.User;
+import org.springframework.security.core.userdetails.UserDetailsService;
+
+public interface UserService extends UserDetailsService {
```

Comment:
Let's put this service on hold and see how we will proceed with the authentication and registration once we have the full web flow.